styles/equipmentSection.scss

```diff
@@ -0,0 +1,109 @@
+.equipment-section {
+  display: flex;
+  width: 100%;
+  width: 100vw;
+  flex-flow: column nowrap;
+
+  padding-top: 100px;
+  padding-bottom: 100px;
+  align-items: center;
+
+  background-color: #2980B9;
+
+    .equipment-header {
+    display: flex;
+    width: 100%;
+    width: 100vw;
+    flex-flow: column nowrap;
+    margin-bottom: 100px;
+
+    h1 {
+      color: white;
+      text-align: center;
+    }
+  }
+
+  .equipment-cards {
+    display: flex;
+    width: 60%;
+    width: 60vw;
+    flex-flow: row wrap;
+
+    margin-bottom: 30px;
+    justify-content: space-around;
+    text-align: left;
+
+    .equipment-card {
+      display: flex;
+      flex-flow: column nowrap;
+      background-size: contain;
```

Comment:
It's good this is explicit but eventually we should use shorthand for background